neurons/miners/StableMiner/base.py

```diff
@@ -346,18 +343,18 @@ async def generate_image(self, synapse: ImageGeneration) -> ImageGeneration:
             if any(self.nsfw_image_filter(images)):
                 logger.info("An image was flagged as NSFW: discarding image.")
                 self.stats.nsfw_count += 1
-                synapse.images = []
+                images = [empty_image_tensor()]
```

Comment:
```suggestion
                images = [empty_image_tensor() for image in synapse.images]
```

Could be nicer because then we're dealing with actual arrays, allowing multi-return is nicer perhaps